packages/just-scripts/src/tasks/webpackCLITask.ts

```diff
@@ -0,0 +1,57 @@
+import { logger, TaskFunction } from "just-task";
+import { tryRequire } from "../tryRequire";
+
+/**
+ * webpackCliInitTask - task for webpack-cli init command
+ *
+ * @param  {string} customScaffold? - to pass any webpack-scaffold
+ * @param  {Boolean=false} auto - to pass the --auto flag, which will generate a default webpack.config.js
+ * @returns TaskFunction
+ */
+export function webpackCliInitTask(
+  customScaffold?: string,
+  auto: Boolean = false
+): TaskFunction {
+  return async function webpackCli() {
+    const init = tryRequire("@webpack-cli/init").default;
```

Comment:
Is this a different one than https://www.npmjs.com/package/webpack-cli ?

Reply:
No it's the same.  This is the implementation of webpack cli init command